.github/workflows/benchmarks.yaml

```diff
@@ -0,0 +1,48 @@
+name: benchmarks
+on: pull_request
+
+env:
+  DVC_TEST: "true"
+
+jobs:
+  build:
```

Comment:
It'd be great if we could just keep all of this in the `dvc-bench` itself, with the [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action).

Reply:
For sure! Looking into it...

Reply:
@skshetry One problem here is that composite actions don't support build matrixes https://github.com/actions/runner/issues/810 , so this is going to be a bit less flexible and we will have to run all benchmarks in one single job, which is fine for now with a small selection of benchmarks, but might not be in the long term. So I would probably wait with creating a composite action for now, until we are sure that it will be enough for our use case here. 🙁

Reply:
Why do we need matrix though?

Reply:
@skshetry To run benchmarks in separate jobs, as they might take quite some time once you run a bunch of them. This is not an issue with 2 benchmarks that we have in this PR, but will be once we add some more, especially ones that work with large data.

Reply:
hmm, it's better to just run small benchmarks in PRs. The jobs already take quite a lot of time, which makes me slower.

Reply:
@skshetry Agreed. So looks like it is pretty clear that this is more of a micro-benchmark run for all PRs, but we might need a fuller bigger benchmark run on-demand for some PRs (maybe on a regular basis too, to check for daily/weekly/etc regressions). Will create an action for now.

Reply:
We can run a full suite on PR on demand with comment or `workflow_dispatch` like we do for binary packages builds.